.github/workflows/validate-json-schema.yml

```diff
@@ -1,19 +1,16 @@
 on: [push]
-name: Validate JSON schema
+name: Validate JSON Schema
 jobs:
-  validate_configurations:
-    name: Validate all json files using their own $schema properties
+  test:
```

Comment:
I changed the workflow name a bit, as its now doing more than what the original title covered?